flag.go

```diff
@@ -1005,7 +1007,11 @@ func (f *FlagSet) parseArgs(args []string, fn parseFunc) (err error) {
 			args, err = f.parseShortArg(s, args, fn)
 		}
 		if err != nil {
-			return
+			if f.errorHandling != IgnoreOnError {
```

Comment:
I'm also interested if there is a way to push this error handling into `Parse()` so all of them are handled together (making code maintenance easier/more obvious) but I'm not seeing a great way to do that either...